.github/workflows/automate.yaml

```
@@ -37,16 +37,19 @@ jobs:
           pip install -r requirements.txt
 
       - name: Run script
-        run: python main.py --days-needed 2 # Custom CLI Command
-#LINT.IF .env.example
+        run: python main.py --days-needed 2 PROMPTS_PATH MESSAGE_PATH
```

This command is the same command that you run locally to spin up the server.
```suggestion
        run: python main.py --days-needed 2
```

---

Awesome, thank you. I definitely forgot to update this.